src/helpers/validation.ts

```diff
@@ -1,4 +1,4 @@
-export const errorValueMissing = 'This is an example of an error message for a required form control.';
+export const errorValueMissing = 'This is an example of an error message.';
```

Comment:
does the name of this helper still make sense if we're generalizing the message? is it still attached to the specific case of a missing value?